modules/meeting/app/components/meeting_agenda_items/item_component/show_component.sass

```diff
@@ -26,6 +26,10 @@ $meeting-agenda-item--author-width: 300px
   &--content
     @include text-shortener
 
+  &.closed-meeting
+    grid-template-columns: 20px auto 1fr minmax(auto, $meeting-agenda-item--author-width)
+    grid-template-areas: "drag-handle content duration author" ". notes notes notes notes"
```

Comment:
I see multiple issues with that:
* The `template-areas` definition is invalid, as there are too many columns defined in the second row.
* I think the whole extra definition as well as the modifier is not needed if you change the original defintion if the last column to from `40px` to `fit-content(40px)`. I admit, that I have not tested it in detail, but it would be nice to get rid of the extra stuff.

Reply:
yeah it works correctly